diff --git a/node_modules/@department-of-veterans-affairs/mobile-component-library/src/utils/translation/i18n.ts b/node_modules/@department-of-veterans-affairs/mobile-component-library/src/utils/translation/i18n.ts
index 9207dfc..4548086 100644
--- a/node_modules/@department-of-veterans-affairs/mobile-component-library/src/utils/translation/i18n.ts
+++ b/node_modules/@department-of-veterans-affairs/mobile-component-library/src/utils/translation/i18n.ts
@@ -5,7 +5,7 @@ import i18n from 'i18next'
 import * as enTranslation from './en.json'
 
 const fallbackLanguage = { languageTag: 'en', isRTL: false }
-const defaultLanguage = getLocales()[0] || fallbackLanguage
+const defaultLanguage = fallbackLanguage
 
 export const resources = {
   en: { translation: enTranslation },
