diff --git a/node_modules/lottie-react-native/src/js/index.d.ts b/node_modules/lottie-react-native/src/js/index.d.ts
index ed43140..6dd4883 100644
--- a/node_modules/lottie-react-native/src/js/index.d.ts
+++ b/node_modules/lottie-react-native/src/js/index.d.ts
@@ -32,6 +32,11 @@ declare module "lottie-react-native" {
     replace: string;
   };
 
+  // Animated.AnimatedInterpolation has been converted to a generic type
+// in @types/react-native 0.70. This way we can maintain compatibility
+// with all versions of @types/react-native`
+type AnimatedInterpolation = ReturnType<Animated.Value['interpolate']>;
+
   /**
    * Properties of the AnimatedLottieView component
    */
@@ -50,7 +55,7 @@ declare module "lottie-react-native" {
      * animation will correspondingly update to the frame at that progress value. This
      * prop is not required if you are using the imperative API.
      */
-    progress?: number | Animated.Value | Animated.AnimatedInterpolation;
+    progress?: number | Animated.Value | AnimatedInterpolation;
 
     /**
      * The speed the animation will progress. This only affects the imperative API. The
